tests/integration/internal/stepfuncs/assess_funcs.go

```diff
@@ -89,6 +104,7 @@ func WaitUntilExpectedTracesPresent(
 	tickDuration time.Duration,
 ) features.Func {
 	return func(ctx context.Context, t *testing.T, envConf *envconf.Config) context.Context {
+		fmt.Printf("%s\n", ctx)
```

Comment:
Is it added intentionally?

Reply:
Nah, just for investigation purposes. Need to clean this :)